test/gtest/ucs/test_mpool.cc

```diff
@@ -73,45 +83,37 @@ class test_mpool : public ucs::test {
         return UCS_LOG_FUNC_RC_CONTINUE;
     }
 
-
     static bool
-    check_chunk_elems_num(ucs_mpool_t *mp,
-                          unsigned alloced_elem_num,
-                          unsigned chunk_num,
-                          unsigned *elem_nums)
+    check_chunk_elems_num(const test_mpool_grow_factor_case_t &test_case)
     {
-        ucs_mpool_chunk_t *chunk = mp->data->chunks;
-        std::vector<void*> objs;
-
-        for (unsigned i = 0; i < alloced_elem_num; ++i) {
-            void *obj = ucs_mpool_get(mp);
-            if (obj == NULL) {
-                ADD_FAILURE() << "Could not allocate object [" 
-                              << i
-                              << "] from mpool";
-                break;
+        ucs_mpool_chunk_t *chunk = test_case.mp.data->chunks;
+        unsigned i, num_chunks;
+        unsigned total_num_elems = test_case.mp.data->quota;
```

Comment:
`const`

Reply:
changed accordingly. removed num_chunks which was not necessary, reused "i".